Assets/Scripts/VisualizeAngle.cs

```diff
@@ -11,6 +11,7 @@ public class VisualizeAngle : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
+        this.GetComponent<Transform>().localScale = this.GetComponent<Transform>().localScale * 2;
```

Comment:
I believe Pat removed this line in favor of putting the scale in the game object in the editor, lets take it out of this branch as well